src/material/expansion/expansion-panel-header.scss

```
@@ -92,4 +92,9 @@
       content: '';
     }
   }
+  .mat-expansion-panel-header {
+    border-bottom: 1px solid;
+    border-bottom-left-radius: 0px;
+    border-bottom-right-radius: 0px;
```

```suggestion
    border-bottom-left-radius: 0;
    border-bottom-right-radius: 0;
```
nit: don't need units for zero

---

done.